app/src/main/res/menu/video_detail_menu.xml

```diff
@@ -3,16 +3,10 @@
       xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <item
-        android:id="@+id/menu_item_download"
-        android:icon="?attr/download"
-        android:title="@string/download"
-        app:showAsAction="always"/>
-
-    <item
         android:id="@+id/menu_item_share"
         android:icon="?attr/share"
         android:title="@string/share"
-        app:showAsAction="ifRoom"/>
```

Comment:
As @mauriciocolli [noted](https://github.com/TeamNewPipe/NewPipe/issues/471#issuecomment-300211270) it, the download button should always stay visible, it should not stay on `ifRoom`.